services/blockchain-indexer/methods/dataService/controllers/accounts.js

```diff
@@ -73,8 +73,25 @@ const getLegacyAccountInfo = async params => {
 	};
 };
 
+const getTokensInfo = async params => {
```

Comment:
Please move all the token-related methods to a corresponding token-specific files.